seacrowd/sea_datasets/uit_viic/uit_viic.py

```diff
@@ -0,0 +1,152 @@
+# coding=utf-8
+import json
+import os.path
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_DATASETNAME = "uit_viic"
+_CITATION = """\
+            @InProceedings{10.1007/978-3-030-63007-2_57,
+            author="Lam, Quan Hoang
+            and Le, Quang Duy
+            and Nguyen, Van Kiet
+            and Nguyen, Ngan Luu-Thuy",
+            editor="Nguyen, Ngoc Thanh
+            and Hoang, Bao Hung
+            and Huynh, Cong Phap
+            and Hwang, Dosam
+            and Trawi{\'{n}}ski, Bogdan
+            and Vossen, Gottfried",
+            title="UIT-ViIC: A Dataset for the First Evaluation on Vietnamese Image Captioning",
+            booktitle="Computational Collective Intelligence",
+            year="2020",
+            publisher="Springer International Publishing",
+            address="Cham",
+            pages="730--742",
+            abstract="Image Captioning (IC), the task of automatic generation of image captions, has attracted
+            attentions from researchers in many fields of computer science, being computer vision, natural language
+            processing and machine learning in recent years. This paper contributes to research on Image Captioning
+            task in terms of extending dataset to a different language - Vietnamese. So far, there has been no existed
+            Image Captioning dataset for Vietnamese language, so this is the foremost fundamental step for developing
+            Vietnamese Image Captioning. In this scope, we first built a dataset which contains manually written
+            captions for images from Microsoft COCO dataset relating to sports played with balls, we called this dataset
+            UIT-ViIC (University Of Information Technology - Vietnamese Image Captions). UIT-ViIC consists of 19,250
+            Vietnamese captions for 3,850 images. Following that, we evaluated our dataset on deep neural network models
+            and did comparisons with English dataset and two Vietnamese datasets built by different methods. UIT-ViIC
+            is published on our lab website (https://sites.google.com/uit.edu.vn/uit-nlp/) for research purposes.",
+            isbn="978-3-030-63007-2"
```

Comment:
Similar to https://github.com/SEACrowd/seacrowd-datahub/pull/255 let's remove the leading whitespace here for consistency with other implementations